tests/ft_php.vader

```diff
@@ -46,3 +46,25 @@ Execute (phpmd):
   AssertEqual loclist[0].text, "Avoid unused parameters such as '$request'."
   AssertEqual loclist[1].lnum, 48
   AssertEqual loclist[1].text, "Avoid variables with short names like $id. Configured minimumlength is 3."
+
+Execute (phpmd):
+  let phpstan_output =
+  \" ------ ----------------------------------------------------------------------------------------------\n
+  \  Line   TaskRepository.php\n
+  \ ------ ----------------------------------------------------------------------------------------------\n
+  \  25     Parameter $taskId of method TodoBundle\Repository\TaskRepository::findById() has invalid typehint type TodoBundle\Repository\Identifier.\n
+  \  27     Call to an undefined method TodoBundle\Repository\TaskRepository::findOneById().\n
+  \ ------ ----------------------------------------------------------------------------------------------\n
+  \\n
+  \ [ERROR] Found 2 errors"
```

Comment:
Is there an easier to parse output formatter maybe, where the file is displayed on each line?
Otherwise it might be possible still, see `:h errorformat-separate-filename`.